integration_test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -Eemo pipefail
+# This script is meant to be run inside https://hub.docker.com/_/docker
+# See .github/workflows/ci.yml:18 for how to run this yourself
+
+# Wait for the docker daemon to start up
+for i in {1..30}; do
+    if docker ps; then
+        break
+    else
+        sleep 1
+    fi
+done
+if [ "$i" = '30' ]; then
+    echo "Docker did not start within 30 seconds, aborting"
+    exit 1
+fi
+
+# Start docuum in the background
+/artifacts/docuum-x86_64-unknown-linux-musl --threshold=13MB &
+sleep 1
```

Comment:
Is this `sleep` necessary, even with the loop below?

Reply:
It was needed because when bash first spawns the Docuum process, the process name is still `./integration_test.sh` for a short period before it changes to `docuum-x86_64-unknown-linux-musl`, which would cause `pgrep` to return nothing and kill the script.

But now that I'm using `$!` to get the pid, it can be removed